### DISCUSSION TITLE
How do I toggle accessorries with three-vrm?

### DISCUSSION THREAD

**Discussion by oz123** [pixiv/three-vrm#1109] [Help]
My model has two Shape Keys, which can be closed or open  (values 0-1) which I can toggle using blender.
However, I could not find an obvious way to do this using this library. Can  you maybe give me a hint?

**Comment by 0b5vr** [comment]
There is a feature called "BlendShapeProxy (in VRM0.0)" or "Expressions (in VRM1.0)" in VRM, which enables you to control morphs.

If your model has the feature, you should be able to control this using [VRMExpressionManager](https://pixiv.github.io/three-vrm/packages/three-vrm/docs/classes/VRMExpressionManager.html) like this:

```js
vrm.expressionManager.setValue("<name of your morph>", <target value in 0-1>);
```

See also the example:

https://pixiv.github.io/three-vrm/packages/three-vrm/examples/expressions.html
https://github.com/pixiv/three-vrm/blob/dev/packages/three-vrm-core/examples/expressions.html
